File: Cms.NetCore/Cms.NetCore.Admin/wwwroot/js/operationalLog/operationalLogList.js
layui.use([ 'jquery', 'layer', 'table'], function () {
    var

        //layer = parent.layer === undefined ? layui.layer : top.layer,
        $ = layui.jquery,
        table = layui.table;

    //用户列表
    var tableIns = table.render({
        elem: '#operationalLogList',
        url: '/operationalLog/GetList',
        cellMinWidth: 95,
        page: true,
        height: "full-20",
        limits: [10, 15, 20, 25],
        limit: 20,
        cols: [[
            { type: "numbers", fixed: "left", width: 50 },
            { field: 'operationalName', title: '功能名称', minWidth: 100, align: "center" },
            { field: 'controller', title: '控制器', minWidth: 100, align: "center" },
            {
                field: 'action', title: '方法', minWidth: 100, align: "center"
            },
            { field: 'operationalTime', title: '操作时间', minWidth: 200, align: "center" },
            {
                field: 'operationalIp', title: 'IP', minWidth: 100, align: "center"
            },
            {
                field: 'operationalState', title: '状态', minWidth: 100, align: "center" ,templet: function (d) {
                    var state = d.operationalState === 1 ? "正常" : "异常"; 
                    var calss = d.operationalState === 1 ? "layui-bg-green" : "layui-bg-red";
                    return '<i class=' + calss+'>' + state + '</i>';
                } },
            { field: 'userName', title: '操作人', align: 'center', minWidth: 100 }
        ]]
    });
})

File: Cms.NetCore/Cms.NetCore.ViewModels/Results/OperationalLog/OperationalLogList.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Cms.NetCore.ViewModels.Results.OperationalLog
{
    public class OperationalLogList
    {
        public Guid Id { get; set; }
        /// <summary>
        /// 操作页面
        /// </summary>
        public string Controller { get; set; }
        /// <summary>
        /// 操作的按钮
        /// </summary>
        public string Action { get; set; }
        /// <summary>
        /// 功能名称
        /// </summary>
        public string OperationalName { get; set; }
        /// <summary>
        /// 操作人
        /// </summary>
        public string UserName { get; set; }
        /// <summary>
        /// 操作时间
        /// </summary>
        public DateTime OperationalTime { get; set; }
        /// <summary>
        /// 操作Ip
        /// </summary>
        public string OperationalIp { get; set; }

        /// <summary>
        /// 操作状态(1：成功，2：异常)
        /// </summary>
        public int OperationalState { get; set; }
    }
}
